src/lib/Scenes/Inbox/Components/Conversations/ConversationCTA.tsx

```diff
@@ -13,21 +13,21 @@ interface Props {
 }
 
 export const ConversationCTA: React.FC<Props> = ({ conversation, show }) => {
-  // Determine whether we have a conversation about an artwork
-  const firstItem = conversation?.items?.[0]?.item
-  const artwork = firstItem?.__typename === "Artwork" ? firstItem : null
-  const artworkID = artwork?.artworkID
+  const liveArtwork = conversation?.items?.[0]?.liveArtwork
+  const artwork = liveArtwork?.__typename === "Artwork" && liveArtwork
+  const isOfferableFromInquiry = !!artwork && !!artwork?.isOfferableFromInquiry
```

Comment:
This also seems redundant:
```suggestion
  const isOfferableFromInquiry = !!artwork?.isOfferableFromInquiry
```

Reply:
This is feasible if I use the ternary operate that I mentioned before. Otherwise artwork could evaluate to `false` and we'd get the type error that `isOfferableFromInquiry` doesn't exist on artwork.